packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -107,4 +112,121 @@ storiesOf('ListView', module)
     <ListView width="300px" height="300px" isLoading>
       {[]}
     </ListView>
-  ));
+  ))
```

Comment:
Should the above stories be updated to have the Content wrapper or something? The buttons are not right aligned for example